UserService/app/src/main/java/com/bookmyshow/userservice/dao/entity/Permission.java

```diff
@@ -0,0 +1,60 @@
+package com.bookmyshow.userservice.dao.entity;
+
+import javax.persistence.*;
+import java.util.List;
+import java.util.Set;
+
+@Table(name = "PERMISSION")
+@Entity
+public class Permission {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "PERMISSION_CODE")
+    private String permissionCode;
```

Comment:
Can this be renamed as "code" as it is anyway inside Permission class. Also, pls write class and variable comments for understanding.